modin/pandas/utils.py

```diff
@@ -238,5 +252,49 @@ def check_both_not_none(option1, option2):
     return not (option1 is None or option2 is None)
 
 
+def _doc_binary_op(
+    operation, bin_op, other="right", returns="Series", primary_object="Series"
+):
+    """
+    Return callable documenting `Series` or `DataFrame` binary operator.
+
+    Parameters
+    ----------
+    operation : str
+        Operation name.
+    bin_op : str
+        Binary operation name.
+    other : str, default: 'right'
+        The second operand name.
+    returns : str, default: 'Series'
+        Type of returns.
+    primary_object : str, default: 'Series'
+        The primary object to document.
```

Comment:
```suggestion
    left : str, default: 'Series'
        The left object to document.
    right : str, default: 'right'
        The right operand name.
    returns : str, default: 'Series'
        Type of returns.
```
left and right look better to me.